analyses/fusion_filtering/run_fusion_merged.sh

```diff
@@ -96,17 +95,12 @@ Rscript 02-fusion-filtering.R --standardFusionFiles $standard_starfusion_file,$s
                               --outputFile "${scratch_path}/standardFusionExp" \
                               --readthroughFilter
 
-# Fusion zscore annotation for filtered fusion for polya
+
+# Fusion zscore annotation for filtered fusion for the combined RNA expression file
 Rscript 03-Calc-zscore-annotate.R --standardFusionCalls "${scratch_path}/standardFusionPolyaExp_QC_expression_filtered_annotated.RDS" \
                                   --expressionMatrix $polya_expression_file \
                                   --normalExpressionMatrix $normal_expression_file \
-                                  --outputfile "${scratch_path}/standardFusionPolyaExp_QC_expression"
-
-# Fusion zscore annotation for filtered fusion for stranded
-Rscript 03-Calc-zscore-annotate.R --standardFusionCalls "${scratch_path}/standardFusionStrandedExp_QC_expression_filtered_annotated.RDS" \
-                                  --expressionMatrix $stranded_expression_file \
-                                  --normalExpressionMatrix $normal_expression_file \
-                                  --outputfile "${scratch_path}/standardFusionStrandedExp_QC_expression"
+                                  --outputFile "${scratch_path}/standardFusionPolyaExp_QC_expression"
```

Comment:
Looks like this should not be polyA-specific?

Reply:
Thanks for pointing that out! Fixed now. Sorry I went back at one point to fix all run bash scripts to match the PRs (with only changes related to the PRs) and missed that.

Reply:
no problem!